command/riff_invoker.go

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2018 The original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package command
+
+import (
+	"fmt"
+	"github.com/buildpack/libbuildpack"
+	"github.com/cloudfoundry/libjavabuildpack"
+	"github.com/projectriff/riff-buildpack"
+	"os"
+	"path/filepath"
+)
+
+const (
+	// RiffCommandInvokerDependency is a key identifying the command invoker dependency in the build plan.
+	riffCommandInvokerDependency = "riff-invoker-command"
+
+	// command is the key identifying the command executable in the build plan.
+	command = "command"
```

Comment:
This is public enough (it's floating around in the `BuildPlan`) that it should be capitalized and made public.